packages/flutter_tools/templates/cocoapods/Podfile-ios-objc

```diff
@@ -1,3 +1,6 @@
+# CocoaPods 1.7.2 brings with it the final version of CDN support for the trunk specs repo.
```

Comment:
Is there some better way we can describe this? I don't think a casual Flutter developer would know what "CDN support for the trunk specs repo" means.
The comment should answer the question: why should I uncomment this? From my reading of the feature it would be to save disk space, but you may have deeper insight!

Reply:
Certainly, I tried to correct the comment.  https://github.com/flutter/flutter/pull/36237/commits/1f9b1c047cfedb66f7890607f5ac43504427ba42